### PR TITLE
Added Note About General Navigability

### LINKED ISSUE #160: Add Note About General Navigability
Split from #128:

> Such mechanisms may include site specific search engines, site outlines, tree diagrams, and the use of proper HTML techniques such as tagging elements as tables, headers, lists and forms.

### PR DESCRIPTION
**Pull request by loganfranken**
Fixes #160

### CODE REVIEW

File: web-accessibility.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ In addition to the PDF above and the recommendations described throughout the We
...
 addressing accessibility in your websites:
 
+#### Help Users Navigate
+
+Ensure that your website is easy for your users to navigate by establishing a
+clearly-defined information architecture and providing straightforward mechanisms for users to
+navigate within this architecture, including search engines and site outlines.
+
+Furthermore, ensure that users can easily orient themselves within the structure of
+website using techniques such as labeling the current section and using
```
Comment (garster) [inline web-accessibility.md]: Should this be "within the structure of **the** website"?

**Comment by loganfranken** [comment]
Hey there! Anything needed from me to merge this?

**Comment by loganfranken** [comment]
Good morning! Need anything from me to merge this?

**Review by garster — CHANGES_REQUESTED** [review]

**Comment by loganfranken** [comment]
Nice catch! Updated!

**Pull request merged**